apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -205,9 +205,13 @@ export class SendMessageEmail extends SendMessageBase {
       to: email,
       subject,
       html,
-      from: command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
+      from:
+        overrides?.email?.from || command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
       attachments,
+      text: overrides?.email?.text,
```

Comment:
Does this fix any bug or it is a leftover from other task?

Reply:
we need to pass the override down if we like it to be sent in the providers